### ISSUE TITLE
Tracking Issue: Fix Sqlsmith workarounds

### ISSUE THREAD

**Issue by kwannoel** [singularity-data/risingwave#3896]
Several features are currently not fully supported, hence some parts of sqlsmith are disabled, or use workarounds. We should re-enable them when their dependent issues are fixed.

- [ ] Enable equijoin. Requires fix: #2771. Previous discussion: https://github.com/singularity-data/risingwave/pull/3613#issuecomment-1180786480.
- [ ] Enable gen subqueries.
- [ ] Enable general `gen_from` for materialized view. Currently requires some workarounds for unsupported executors.

Related: #2571

**Comment by kwannoel** [comment]
Some updates on what I intend to do:
- In order to keep progress of sqlsmith moving, I may implement some features, but disable in PR if they are blocked by some bugs.
- Then, most of the work that will be left is to enable / disable sqlsmith features as bugs get fixed.

We can have a separate issue to track:
1. nice-to-haves e.g. test suite usability refinements
2. enable / disable features.

Then these will be lower priority.
